### PR TITLE
gcovr: add missing runtime dependency

### PR DESCRIPTION
**Pull request by unspecd**
Fixes: `ModuleNotFoundError: No module named 'lxml'`.

### CODE REVIEW

**Comment by ericonr** [comment]
Could you add `checkdepends="$depends python3-pytest"`, please?

**Comment by unspecd** [comment]
> Could you add `checkdepends="$depends python3-pytest"`, please?

I am getting the following error:
```
$ ./xbps-src -Q pkg gcovr
…
ModuleNotFoundError: No module named 'pyutilib'
```

File: srcpkgs/gcovr/template [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Template file for 'gcovr'
 pkgname=gcovr
 version=4.2
-revision=3
+revision=4
 build_style=python3-module
-pycompile_module="gcovr"
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-Jinja2"
+depends="python3-setuptools python3-Jinja2 python3-lxml"
```
Comment (ericonr) [inline srcpkgs/gcovr/template:7]: You should add pygments to depends: https://github.com/gcovr/gcovr/blob/master/setup.py#L26 (the proper void package name, of course).

File: srcpkgs/gcovr/template [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+checkdepends="$depends python3-pytest"
```
Comment (ericonr) [inline srcpkgs/gcovr/template:8]: And probably all these: https://github.com/gcovr/gcovr/blob/master/requirements.txt to checkdepends.

**Comment by unspecd** [comment]
I give up.

**Comment by ericonr** [comment]
Please don't :(

What are you stumped on? I can try to pick it up, but I don't use the package myself.

**Comment by unspecd** [comment]
> What are you stumped on?

I can't figure out why these errors occur:
```sh
$ cat masterdir/builddir/.xbps-gcovr/gcovr__do_check.log | grep 'not found'
/builddir/gcovr-4.2/doc/examples/example.sh: line 16: gcovr: command not found
/builddir/gcovr-4.2/doc/examples/example_branches.sh: line 8: gcovr: command not found
/builddir/gcovr-4.2/doc/examples/example_xml.sh: line 8: gcovr: command not found
```

**Comment by ericonr** [comment]
Sorry for the delay here :(

**Pull request closed without merging**